app/controllers/v0/claim_letters_controller.rb

```diff
@@ -24,7 +24,18 @@ def show
     private
 
     def service
-      @service ||= ClaimStatusTool::ClaimLetterDownloader.new(@current_user)
+      @service ||= ClaimStatusTool::ClaimLetterDownloader.new(@current_user, allowed_doctypes)
+    end
+
+    # 27: Board Of Appeals Decision Letter
+    # 184: Notification Letter (e.g. VA 20-8993, VA 21-0290, PCGL)
+    # 339: Rating Decision Letter
+    def allowed_doctypes
+      if Flipper.enabled?(:cst_include_ddl_boa_letters, @user)
+        %w[27 184]
+      else
+        %w[184]
+      end
```

Comment:
So on the existing route this PR will leave the doc types the same, right?

Reply:
After this PR you will have another PR for updating doc types internally. And before that PR, we need update our bot skill to point to the new route from below. right?

Reply:
@ksk385 The existing route and controller will still work the way it was before, so we don't need any additional PRs. There is a new controller (`app/controllers/v0/virtual_agent_claim_letters_controller.rb`) and route (`config/routes.rb:158`) in this PR for your use when you want to switch your skill over to start using it. I don't see any need to have multiple PRs given how this is structured. Thoughts?

Reply:
@ksk385 I understand your concern here, but to clarify: the changes to the existing controller are merely a refactor and decoupling of where the control over allowed_doctypes exists. It doesn't actually change any of the existing functionality, but sets up the ability to independently control allowed_doctypes from different controllers. It won't affect how the existing route works or what it returns in the short term. Does that make sense? cc: @jerekshoe

Reply:
Yes makes sense! Thank you!